lib/nexmo_developer/app/views/static/landing_page_documentation.html.erb

```diff
@@ -2,14 +2,26 @@
   <div class="center spacious content-block">
     <h1><%= @landing_info['title'].html_safe %></h1>
     <p><%= raw @landing_info['subtitle'].html_safe %></p>
-  </div>
+    <p>
+      <!-- This is the buttons group area-->
+      <strong>Jump to: </strong>
+      <% @landing_info['top-section'].each do |product| %>
+        <a href="#<%=product.keys.first.parameterize %>"  class='Vlt-btn Vlt-btn--blue Vlt-btn--app Vlt-btn--small'><%= product.keys.first %></a>
+      <%end %>
 
-  <div>
+      <% @landing_info['bottom-section'].each do |product| %>
+        <% product[product.keys.first].each do |cards| %>
+          <a href="#<%=cards.keys.first.parameterize %>" class='Vlt-btn Vlt-btn--blue Vlt-btn--app Vlt-btn--small'><%= cards.keys.first %></a>
```

Comment:
same please

```
Vlt-btn Vlt-btn--primary Vlt-btn--small

Reply:
I thought Nick said to make it tertiary. Ill change it to primary and we can discuss tomorrow.